packages/charts/react-charting/src/utilities/SVGTooltipText.tsx

```diff
@@ -140,14 +140,14 @@ export class SVGTooltipText
     const showTooltip =
       (!!this.props.isTooltipVisibleProp && this.state.isOverflowing && !!content) || (isTooltipVisible && !!content);
     const backgroundColor = this.props.theme ? this.props.theme.semanticColors.bodyBackground : undefined;
-    const isRTL = getRTL();
```

Comment:
>isRTL [](http://example.com/codeflow?start=10&length=5)

rtl is not needed anymore?

Reply:
`rect`'s x and y attributes depend on the `text`'s x and y, which have already been adjusted based on the layout direction. So, there is no need to check for RTL again when positioning the `rect`.